apps/bc-wallet-api-server/.env.example

```diff
@@ -15,3 +15,9 @@ ALLOW_ORIGINS=https://bcshowcase-ui.dev.nborbit.ca,http://localhost:3000
 ALLOW_METHODS=GET,POST,PUT,DELETE,OPTIONS
 ALLOW_HEADERS=Content-Type,Authorization,X-Requested-With,Cache-Control,Pragma,Expires
 ALLOW_CREDENTIALS=true
+
+# authentication & authorization
+REALM=BC
+AUTH_SERVER_URL=http://localhost:8080
+CLIENT_ID=showcase-tenantA
+CLIENT_SECRET=kFY2iBqHHYqLtaQ0PjYrc7eGkYvrasaU
```

Comment:
Maybe better =YOUR_CLIENT_SECRET in Github

Reply:
Removed the actual data from .env.example file